restbridge/detail/RequestHandler.cpp

```diff
@@ -107,7 +106,6 @@ void RequestHandler::onStartupHeartbeatEvent_()
 
 void RequestHandler::onHeartbeatEvent_()
 {
-    LBVERB << "Heartbeat event received." << std::endl;
 }
```

Comment:
The method is used in onVocabularyEvent_ and should not be removed. It will later be used for getting the status of the application.